servicetalk-grpc-netty/src/test/java/io/servicetalk/grpc/netty/ErrorHandlingTest.java

```diff
@@ -437,9 +439,14 @@ private void setupForBlockingServiceWritesThenThrows(final BlockingTesterService
     @AfterEach
     void tearDown() throws Exception {
         try {
-            blockingClient.close();
+            if (null != blockingClient) {
+                blockingClient.close();
+            }
         } finally {
-            newCompositeCloseable().appendAll(client, serverContext).close();
+            newCompositeCloseable().appendAll(Stream.of(client, serverContext)
+                    .filter(Objects::nonNull)
+                    .toArray(AsyncCloseable[]::new)
+            ).close();
```

Comment:
The test was throwing NPE without actually closing the resources because some or all of the references were null. I only noticed because I had enabled a debugger breakpoint for uncaught exceptions.